<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mapp</class>
 <widget class="QWidget" name="mapp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>672</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QRadioButton" name="place1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>460</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>39楼</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="place2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>410</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>家园食堂</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="place5">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>230</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>图书馆</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="place3">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>460</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>五四操场</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="place4">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>330</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>机房</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="place6">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>50</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>未名湖</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>0</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbme">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>69</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>lbme</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbop">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>200</y>
     <width>69</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>lbop</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>51</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Lv.</string>
   </property>
  </widget>
  <widget class="QSlider" name="exp">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lbusername">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>69</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>username</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
